### PR TITLE
feat: Language selection dropdown

### LINKED ISSUE #104: FeedAHelp: Add a Translation dropdown Icon
Blocked By:  #95 

Image

### PR DESCRIPTION
**Pull request by sojib-mondol**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #104 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx [hunk 1]
```diff
@@ -26,6 +30,10 @@ const NewNav = () => {
             <img className="center ring" alt="" />
             <img className="inner ring" alt="" />
           </Styled.Avatar>
+         
+          {/* Dropdown  langunage list*/}
+          <TranslationDropdown/>
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx]: rename this to `<LanguageSelections />`

File: apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx [hunk 2]
```diff
@@ -26,6 +30,10 @@ const NewNav = () => {
             <img className="center ring" alt="" />
             <img className="inner ring" alt="" />
           </Styled.Avatar>
+         
+          {/* Dropdown  langunage list*/}
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx]: delete this comment

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
You PR Heading is wring format:
image

Rename to: `feat: Language selection dropdown`

File: apps/feedahelp/src/components/translationDropdown/TranslationDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import language from "./language.json";
+
+
+const TranslationDropdown: React.FC = () => {
+
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/translationDropdown/TranslationDropdown.tsx]: try this,
```
const [isOpen, setIsOpen] = useState(false);

    const toggleDropdown = () => {
        setIsOpen(prev => !prev);
    };

    return (
        <div className='relative flex flex-col items-center rounded-lg'>
            <button
                onClick={toggleDropdown}
                className={`bg-gray-300 p-2 w-full flex items-center justify-center font-bold rounded-lg tracking-wider border-4 border-transparent active:border-white duration-300 ${
                    isOpen ? 'active:text-white' : ''
                }`}
            >
                Languages
                <p className='pl-2'>{isOpen ? '↑' : '↓'}</p>
            </button>
            {isOpen && (
                <div className='bg-gray-300 top-20 flex flex-col items-start rounded-lg p-2 w-full absolute'>
                    {language.map((item, i) => (
                        <div
                            key={i}
                            className='p-2 hover:bg-slate-400 w-full flex cursor-pointer rounded-r-lg border-l-transparent hover:border-l-white border-l-4'
                        >
                            <h3 className='font-bold'>{item?.language}</h3>
                        </div>
                    ))}
                </div>
            )}
        </div>
    );
    ```

File: apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx (hunk 1 above)
Comment (sojib-mondol) [reply apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx]: Resolved

File: apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx (hunk 2 above)
Comment (sojib-mondol) [reply apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx]: Resolved

File: apps/feedahelp/src/components/translationDropdown/TranslationDropdown.tsx (hunk 3 above)
Comment (sojib-mondol) [reply apps/feedahelp/src/components/translationDropdown/TranslationDropdown.tsx]: Resolved

**Review by TanvirAlam — DISMISSED** [review]
LGTM, lets merge we will fix this when time comes,
nice work @sojib-sudo

**Review by TanvirAlam — DISMISSED** [review]

**Review by TanvirAlam — APPROVED** [review]

**Pull request merged**